week02/bom.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
You didn't finish this activity! Please follow the instructions for the activity and let me know if you need any help.

These are the two activities you need to refer to in order to finish it:

- [Prepare: DOM Manipulation](https://byui-cse.github.io/wdd131-ww-course/week02/prepare-dom-manipulation.html)
- [Prepare: DOM Event Handling](https://byui-cse.github.io/wdd131-ww-course/week02/prepare-dom-eventhandling.html)